.github/workflows/ci.yaml

```diff
@@ -59,5 +59,9 @@ jobs:
         if: steps.list-changed.outputs.changed == 'true'
         run: make crds
 
+      - name: Wait
```

Comment:
There's a race condition and without this wait `ct` can't see the flink operator CRD